.github/ISSUE_TEMPLATE/misc.yml

```diff
@@ -0,0 +1,18 @@
+name: Issue
+description: General Issue. For usage questions, please follow the "Usage question" link
+title: '[MISC]: '
```

Comment:
```suggestion
```
I prefer to leave the title open so it does not end up as `[MISC]:`

Reply:
We need some sort of a placeholder because issue forms require a title (I believe empty strings aren't allowed).

Reply:
[newfeature.yml](https://raw.githubusercontent.com/pydata/xarray/main/.github/ISSUE_TEMPLATE/newfeature.yml) doesn't seem to contain a `title` - maybe it can be left out entirely?

Reply:
Good catch... Leaving it out should suffice